proposer/config.go

```diff
@@ -69,11 +64,11 @@ func NewConfigFromCliContext(c *cli.Context) (*Config, error) {
 		proposeEmptyBlocksInterval = &interval
 	}
 
-	localAddresses := []common.Address{}
+	var localAddresses []common.Address
 	if c.IsSet(flags.TxPoolLocals.Name) {
 		for _, account := range strings.Split(c.String(flags.TxPoolLocals.Name), ",") {
 			if trimmed := strings.TrimSpace(account); !common.IsHexAddress(trimmed) {
-				return nil, fmt.Errorf("invalid account in --txpool.locals: %s", trimmed)
+				return nil, fmt.Errorf("invalid account in --txpool.LocalAddresses: %s", trimmed)
```

Comment:
```suggestion
				return nil, fmt.Errorf("invalid account in --txpool.locals: %s", trimmed)
```

Looks like the flag name is not changed, so we should still print `--txpool.locals` here.